### ISSUE TITLE
Suggestion: change TeleportEndPoints to link to teleporter instead of base computer

### ISSUE THREAD

**Issue by Catsbuggy** [goatfungus/NMSSaveEditor#104]
allows to change the TeleportEndPoints of the player's bases by automatically placing them at the Base Teleport Module of the base.

sorry for my very bad english i hope is understandable.

**Comment by goatfungus** [comment]
That is a good suggestion.

I am hoping HG will correct this in the game itself as it makes more sense to have the teleporter endpoint next to your teleporter, but instead it spits you out on top of your base computer. It really shouldn't be that hard to fix this in-game.

**Comment by Catsbuggy** [comment]
happy you like this suggestion, I'm looking forward to having this feature in my favorite save editor :)

thanks you in advance

**Comment by goatfungus** [comment]
From my testing, there looks to be a bug in the game when generating the coordinates for the teleport endpoint. The coordinates for many of my base teleports seem to be miles away from the base itself, and sometimes round the other side of the planet!

Rather than fixing this bug, it appears that HG have opted to ignore these generated coordinates and set it to your base computer when you initiate the teleport. This was probably the easiest solution for them at the time to avoid them having to spend time calculating the coordinates properly, but it is not ideal. It means we probably can't change the coordinates in the save file because the game will ignore them anyway.

**Comment by Catsbuggy** [comment]
really sad new :'(
I look forward to this improvement with great expectation, thank you very much goatfungus for trying

**Comment by goatfungus** [comment]
I'll keep this one open and periodically check for solutions.

**Comment by Goose7** [comment]
Your MOVE BASE feature proved to be a GodSend for me even though it isn't the perfect solution.

I found that as you built out your base, the spot the game would decide to spit you out when you Teleported there would change over time and often times get more and more bizarre.

Just about the time you added the MOVE feature, I was at my wits end!  Some of my mountain top bases had become totally FUBAR!  I would first appear right in front of the base, I assume standing on top of the Base Computer.  The image would be wavering forward and back in a strange way the whole time, while the Planet Info screen was being displayed.

As soon as this overlay faded out, my character would literally be flung over the side of a cliff, and I would go sailing down onto the ground, rocks and/or ocean below!  It was really a stupid problem to be forced to be dealing with!

I found through a lot of trial and error, that moving the Base Computer a few feet forward, backward and/or left to right could change this behavior and prevent my guy from getting launched.

The other issue is where your ship gets placed.  I would mark out this location at a new base site and expect the ship to always be placed there in the future.  This too would not be the case, and often times, my ship would start getting spawned in some unreachable area.  Buried into the side of the mountain or some other equally DUMB location.

But thanks to your MOVE BASE feature, I was at least able to exert some control over this stupid process and eventually get all my bases to work in an acceptable manner when spawning in to them.

Being able to use the teleporter to determine where you end up would again be a much better solution, but as I see from your research, HG made sure that was likely never going to be an option.
